.github/workflows/master.yml

```diff
@@ -1,6 +1,6 @@
 env:
   AZDO_ORG_SERVICE_URL: ${{ secrets.AZDO_ORG_SERVICE_URL }}
-  AZDO_PERSONAL_ACCESS_TOKEN: ${{ secrets.AZDO_PERSONAL_ACCESS_TOKEN }}
+  AZDO_PERSONAL_ACCESS_TOKEN: ${{ secrets.AZDO_ORG_SERVICE_URL }}
```

Comment:
With this change the provider won't be able to authenticate against the Azure DevOps REST API, because the wrong environment variable is expanded `AZDO_ORG_SERVICE_URL`.